README.md

```diff
@@ -5,11 +5,9 @@
     <img src="README_resources/classiq-logo.svg" width="300" height="150">
 </div>
 
-# Classiq
+# Classiq: Quantum Programming Made Simple
```

Comment:
Hello @Smoothengineer! 
Thanks for your PR with these suggestions :)

Could you please change this line to: "Classiq: High-Level Quantum Modeling Language"

Reply:
sure

Reply:
Sure sir.